google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -1003,6 +1003,19 @@ public Acl blobToPublicRead(String bucketName, String blobName, long blobGenerat
     return buckets;
   }
 
+  /** Example of a simple listBuckets() */
+  public Page<Bucket> listBuckets() {
+    // [START storage_list_buckets]
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    Page<Bucket> buckets = storage.list();
+    for (Bucket bucket : buckets.iterateAll()) {
+      // do something with the info
+    }
+    // [END storage_list_buckets]
+    return buckets;
```

Comment:
Or why do we return and iterate the buckets? Feels like we should either use the value or return it, not both.